column_transforms/rasgo_substring/rasgo_substring.sql

```diff
@@ -0,0 +1,10 @@
+SELECT
+*
+{%- for col_expr, start_pos, end_pos in new_cols %}
```

Comment:
do we anticipate needing to run multiple substrings on multiple different columns in one UDT?